source/type/pb_converter.cpp

```diff
@@ -750,6 +750,16 @@ void PbCreator::Filler::fill_pb_object(const nt::Network* n, pbnavitia::Network*
 void PbCreator::Filler::fill_pb_object(const nt::PhysicalMode* m, pbnavitia::PhysicalMode* physical_mode) {
     physical_mode->set_name(m->name);
     physical_mode->set_uri(m->uri);
+
+    if (depth == 0) {
+        return;
+    }
+
+    if (m->co2_emission) {
+        auto* emission_rate = physical_mode->mutable_co2_emission_rate();
+        emission_rate->set_value(*m->co2_emission);
+        emission_rate->set_unit("gCO2/Km");
```

Comment:
```suggestion
        emission_rate->set_unit("gEC/Km");
```
Since the unit on CO2 emission on a section are gEC, this one should be gEC/Km, no ?